### PR TITLE
PHP 8.4: Document reflection changes

### PR DESCRIPTION
**Pull request by Girgias**
Drive-by fix for 8.1 ommision

### CODE REVIEW

File: reference/reflection/reflectionclass/resetaslazyghost.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <methodsynopsis role="ReflectionClass">
    <modifier>public</modifier> <type>void</type><methodname>ReflectionClass::resetAsLazyGhost</methodname>
    <methodparam><type>object</type><parameter>object</parameter></methodparam>
-   <methodparam><type>callable</type><parameter>initializer</parameter></methodparam>
```
Comment (arnaud-lb) [inline reference/reflection/reflectionclass/resetaslazyghost.xml:13]: This one is a bug in the stub, unfortunately :( The parameter should have been called "initializer".

**Review by arnaud-lb — APPROVED** [review]
Thank you!

File: reference/reflection/reflectionclass/resetaslazyghost.xml (hunk 1 above)
Comment (nielsdos) [reply reference/reflection/reflectionclass/resetaslazyghost.xml:13]: 8.4.0 is not released yet, why wait for 8.4.1?

**Review by Crell — APPROVED** [review]

File: reference/reflection/reflectionclass/resetaslazyghost.xml (hunk 1 above)
Comment (arnaud-lb) [reply reference/reflection/reflectionclass/resetaslazyghost.xml:13]: I would like to, but can we still make changes to 8.4.0? As I understand, the current RC will be released as 8.4.0? Or am I mistaken?

Comment (arnaud-lb) [reply reference/reflection/reflectionclass/resetaslazyghost.xml:13]: Oh, that's a good news. Thank you

File: reference/reflection/reflectionproperty.xml [hunk 2]
```diff
@@ -176,6 +206,55 @@
...
+     <varlistentry xml:id="reflectionproperty.constants.is-final">
+      <term><constant>ReflectionProperty::IS_FINAL</constant></term>
+      <listitem>
+       <simpara>
+        Indicates the property is
+        <link linkend="language.oop5.final">final</link>.
+        Available as of PHP 8.1.0.
+       </simpara>
+      </listitem>
+     </varlistentry>
```
Comment (Girgias) [inline reference/reflection/reflectionproperty.xml:256]: Actually I think I misunderstood this, as this is part of property hooks right @Crell @iluuu1994 ?

Comment (iluuu1994) [reply reference/reflection/reflectionproperty.xml:256]: Yes. Properties may not be final before 8.4.

**Pull request merged**